### PR TITLE
Update tibdex/github version

### PR DESCRIPTION
**Pull request by tuckerzp**
Update the tibdex/github version as well as change some wording

Part of closing https://github.com/EasyDynamics/easygrc/issues/44

### CODE REVIEW

**Review by kylelaker — APPROVED** [review]

File: .github/workflows/triage-pr.yml [hunk 1]
```diff
@@ -69,7 +69,7 @@ jobs:
         # app's elevated permissions as infrequently as possible.
         if: steps.pr-reviews.outputs.is-draft == 'false' && steps.pr-reviews.outputs.review-decision == 'REVIEW_REQUIRED'
         id: generate_token
-        uses: tibdex/github-app-token@586e1a624db6a5a4ac2c53daeeded60c5e3d50fe
+        uses: tibdex/github-app-token@f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06
         with:
           app_id: ${{ secrets.APP_ID }}
           private_key: ${{ secrets.APP_PRIVATE_KEY }}
```
Comment (kylelaker) [inline .github/workflows/triage-pr.yml:75]:
```suggestion
          private_key: ${{ secrets.APP_PRIVATE_KEY }}
          permissions: >-
            {
              "pull_requests": "write"
            }
```

File: .github/workflows/update-dependencies.yml [hunk 2]
```diff
@@ -57,7 +57,7 @@ jobs:
         # an output. It _does_ register that token as a secret so that it will be
         # filtered from log output automatically
         id: generate-token
-        uses: tibdex/github-app-token@586e1a624db6a5a4ac2c53daeeded60c5e3d50fe
+        uses: tibdex/github-app-token@f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06
         with:
           app_id: ${{ secrets.APP_ID }}
           private_key: ${{ secrets.APP_PRIVATE_KEY }}
```
Comment (kylelaker) [inline .github/workflows/update-dependencies.yml:63]:
```suggestion
          private_key: ${{ secrets.APP_PRIVATE_KEY }}
          permissions: >-
            {
              "contents": "write",
              "pull_requests": "write",
            }
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -26,7 +26,7 @@ jobs:
         # an output. It _does_ register that token as a secret so that it will be
         # filtered from log output automatically
         id: generate-token
-        uses: tibdex/github-app-token@586e1a624db6a5a4ac2c53daeeded60c5e3d50fe
+        uses: tibdex/github-app-token@f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06
         with:
           app_id: ${{ secrets.APP_ID }}
           private_key: ${{ secrets.APP_PRIVATE_KEY }}
```
Comment (kylelaker) [inline .github/workflows/release.yml:32]: If we use this version, we should use the new permissions scoping feature that came with it.

https://github.com/tibdex/github-app-token/blob/f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06/action.yml#L15-L16

```suggestion
          private_key: ${{ secrets.APP_PRIVATE_KEY }}
          permissions: >-
            {
              "contents": "write"
            }
```

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -26,7 +26,7 @@ jobs:
         # an output. It _does_ register that token as a secret so that it will be
         # filtered from log output automatically
         id: generate-token
-        uses: tibdex/github-app-token@586e1a624db6a5a4ac2c53daeeded60c5e3d50fe
+        uses: tibdex/github-app-token@f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06
```
Comment (kylelaker) [inline .github/workflows/release.yml:29]: Here (and everywhere), can we add a comment that explains what version this maps to? We should have done that before...

```suggestion
        uses: tibdex/github-app-token@f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06 # v1.6.0
```

Also, feel free to tell me that we should consider just using the `release/v1.6` branch. We had that discussion on the PR where this was originally introduced so before starting a convo, please read the earlier one for context.

**Review by kylelaker — CHANGES_REQUESTED** [review]

**Review by kylelaker — APPROVED** [review]
Nice! This is looking good! Thank you@

File: .github/workflows/triage-pr.yml [hunk 5]
```diff
@@ -69,10 +69,15 @@ jobs:
...
         id: generate_token
-        uses: tibdex/github-app-token@586e1a624db6a5a4ac2c53daeeded60c5e3d50fe
+        # This maps to v1.6.0 https://github.com/tibdex/github-app-token/releases/tag/v1.6.0
+        uses: tibdex/github-app-token@f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06
         with:
           app_id: ${{ secrets.APP_ID }}
           private_key: ${{ secrets.APP_PRIVATE_KEY }}
+          permissions: >-
+            {
+              "pull_requests": "write"
```
Comment (kylelaker) [inline .github/workflows/triage-pr.yml:79]:
```suggestion
              "pull_requests": "write",
              "members": "read",
```

**Review by kylelaker — APPROVED** [review]

**Review by Bronstrom — APPROVED** [review]

**Pull request merged**